core/src/main/java/org/hibernate/ogm/service/impl/OgmIntegrator.java

```diff
@@ -73,10 +72,8 @@ private void attachEventContextManagingListenersIfRequired(SessionFactoryService
 		eventListenerRegistry.addDuplicationStrategy( EventContextManagingFlushEventListenerDuplicationStrategy.INSTANCE );
 		eventListenerRegistry.getEventListenerGroup( EventType.FLUSH ).appendListener( new EventContextManagingFlushEventListener( stateManager ) );
 
-		if ( getIntegrator( JpaIntegrator.class, serviceRegistry ) != null ) {
```

Comment:
One a more general note, do you know why something is attached to the persist lifecycle to begin with? I should know as I wrote it, but it completely slipped my mind. I'm finding the `executeBatch()` in `OperationsQueueLifecycle` a bit weird in the context of persist. Wondering whether that's alright.